### ISSUE TITLE
Crashes on some search terms

### ISSUE THREAD

**Issue by aeroechelon** [ajgon/alfred2-quiver-search#4]
Crashes upon some search terms. Behaviour falls back to Alfred fallback.

```
Starting debug for 'Quiver Search'

[ERROR: alfred.workflow.input.scriptfilter] Code 1: /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/json/common.rb:155:in `encode': "\xC2" on US-ASCII (Encoding::InvalidByteSequenceError)
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/json/common.rb:155:in `initialize'
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/json/common.rb:155:in `new'
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/json/common.rb:155:in `parse'
    from -e:47:in `block in <main>'
    from -e:42:in `map'
    from -e:42:in `<main>'
```

Example

![image](https://cloud.githubusercontent.com/assets/2360095/12931364/5afd95ce-cf4b-11e5-96c9-f598acebc6d5.png)

**Comment by yohasebe** [comment]
Inserting a line of ruby code to "Script" fixed a similar problem. Try this:

```
 Encoding.default_external = "utf-8"
```

![2016-02-10_09-55-03](https://cloud.githubusercontent.com/assets/18207/12935847/67a2e05e-cfdc-11e5-980c-b5d1e54a5ce0.png)

**Comment by aeroechelon** [comment]
@yohasebe Thank you! That worked!

**Comment by ajgon** [comment]
Merged in PR #3
